tests/test_files/logs/eko/208-HERA_CC_318GEV_EM_SIGMARED.log

```diff
@@ -0,0 +1,70 @@
+2022-06-16 11:19:58,734 pineko.theory/INFO: Start computation of HERA_CC_318GEV_EM_SIGMARED
```

Comment:
Are you sure that this had to be commited? @andreab1997

Reply:
Same for the others, actually...

Are you sure that this commit is worth?

Reply:
No sorry, I can remove the logs. But i need the directories

Reply:
Make a fixture that creates the tree